extensions/notebook/src/jupyter/serverInstance.ts

```diff
@@ -209,7 +209,7 @@ export class PerFolderServerInstance implements IServerInstance {
 		}
 		let notebookDirectory = this.getNotebookDirectory();
 		this._token = await utils.getRandomToken();
-		let startCommand = `"${this.options.install.pythonExecutable}" "${this.notebookScriptPath}" --no-browser --no-mathjax --notebook-dir "${notebookDirectory}" --NotebookApp.token=${this._token}`;
+		let startCommand = `"${this.options.install.pythonExecutable}" "${this.notebookScriptPath}" --no-browser --ip=127.0.0.1 --allow-root --no-mathjax --notebook-dir "${notebookDirectory}" --NotebookApp.token=${this._token}`;
```

Comment:
@chlafreniere FYI (although I recall you had already discussed this? Are there any security concerns with doing this?

Reply:
yes, I recall discussing on the previous PR.  I'm planning to see if there is a way to check if we're in web mode from the extension.  or we can add a setting to enable.  likely I'll ask the notebook team to address separately.

Reply:
Thanks @kburtram. https://github.com/microsoft/azuredatastudio/issues/15927 to track.

Reply:
@chlafreniere @Charles-Gagnon I'll spend some time today to see if we have an existing way to determine from the extension host that we are running in webmode, and update as suggested.  Otherwise, we can likely push and follow-up using the issue to address this separately.